README.md

```diff
@@ -1 +1,43 @@
-uProxy Windows Installer
+# uProxy Windows Installer
+
+uProxy is a browser plugin that allows users to reach the open internet through a trusted friend or uProxy cloud server.
+
+The uProxy Windows Installer creates a modified version of the Firefox Windows Installer with the uProxy extension enabled by default. This allows Windows users to get up and running with uProxy quickly.
+
+###Dev setup:
+- Need to use a Windows OS
+- [7-zip](http://www.7-zip.org/) and [signtool](https://www.microsoft.com/en-us/download/details.aspx?id=8279), which is part of Windows SDK 7.1, need to be installed
+
+### To build Windows installer:
+- `git clone https://github.com/uProxy/uproxy-windows-installer.git`
+- `install.bat`
+- Grab executable files from `dist\%language%\Firefox_with_uProxy.exe` and upload them to [releases](https://github.com/uProxy/uproxy-windows-installer/releases). By default we build versions for English and Chinese.
```

Comment:
English (us-En) and Chinese (zh-CN)?  So that the reader doesn't have to look around for the `%language%` code here, or for the arg to `install.bat` below.